internal/api/ui/login/static/i18n/zh.yaml

```diff
@@ -0,0 +1,377 @@
+  Title: 多因素验证
+  Description: 验证您的多因素身份认证。
+  OTPDescription: 使用您的身份验证器应用程序（例如 Google Authenticator）扫描代码或复制密码并在下方插入生成的代码。
+  SecretLabel: 秘钥
+  CodeLabel: 验证码
+  NextButtonText: 继续
+  CancelButtonText: 取消
+
+InitMFAU2F:
+  Title: 多因素身份认证设置 U2F / WebAuthN
+  Description: 通过提供名称添加您的令牌，然后单击下面的“注册令牌”按钮。
+  TokenNameLabel: 令牌/机器的名称
+  NotSupported: 您的浏览器不支持 WebAuthN。请确保它是最新的或使用其他版本（例如 Chrome、Safari、Firefox）
+  RegisterTokenButtonText: 注册令牌
+  ErrorRetry: 重试、创建新挑战码或选择不同的方法。
+
+InitMFADone:
+  Title: 多因素身份认证验证完成
+  Description: 多因素验证成功完成。必须在每次登录时输入多因素身份认证。
+  NextButtonText: 继续
+  CancelButtonText: 取消
+
+MFAProvider:
+  Provider0: OTP（一次性密码）
+  Provider1: U2F（通用第二因素身份认证）
+  ChooseOther: 或选择其他选项
+
+VerifyMFAOTP:
+  Title: 验证多因素
+  Description: 验证您的多因素
+  CodeLabel: 验证码
+  NextButtonText: 继续
+
+VerifyMFAU2F:
+  Title: 多因素验证
+  Description: 验证您的多因素 U2F / WebAuthN 令牌
+  NotSupported: 您的浏览器不支持 WebAuthN。确保您使用的是最新版本或将您的浏览器更改为受支持的版本（Chrome、Safari、Firefox）
+  ErrorRetry: 重试、创建新请求或选择其他方法。
+  ValidateTokenButtonText: 验证令牌
+
+Passwordless:
+  Title: 无密码登录
+  Description: 验证您的令牌
+  NotSupported: 您的浏览器不支持 WebAuthN。请确保它是最新的或使用其他版本（例如 Chrome、Safari、Firefox）
+  ErrorRetry: 重试、创建新挑战码或选择不同的方法。
+  LoginWithPwButtonText: 使用密码登录
+  ValidateTokenButtonText: 验证令牌
+
+PasswordlessPrompt:
+  Title: 无密码登录设置
+  Description: 您想设置无密码登录吗？
+  DescriptionInit: 您需要设置无密码登录，使用您提供的链接来注册您的设备。
+  PasswordlessButtonText: 去设置无密码登录
+  NextButtonText: 继续
+  SkipButtonText: 跳过
+
+PasswordlessRegistration:
+  Title: 无密码设置
+  Description: 通过提供名称添加您的令牌，然后单击下面的“注册令牌”按钮。
+  TokenNameLabel: 令牌/机器的名称
+  NotSupported: 您的浏览器不支持 WebAuthN。请确保它是最新的或使用其他版本（例如 Chrome、Safari、Firefox）
+  RegisterTokenButtonText: 注册令牌
+  ErrorRetry: 重试、创建新挑战码或选择不同的方法。
+
+PasswordlessRegistrationDone:
+  Title: 无密码登录设置完成
+  Description: 已成功添加无密码令牌。
+  DescriptionClose: 您现在可以关闭此窗口
+  NextButtonText: 继续
+  CancelButtonText: 取消
+
+PasswordChange:
+  Title: 更改密码
+  Description: 更改您的密码。输入您的旧密码和新密码。
+  OldPasswordLabel: 旧密码
+  NewPasswordLabel: 新密码
+  NewPasswordConfirmLabel: 确认密码
+  CancelButtonText: 取消
+  NextButtonText: 继续
+
+PasswordChangeDone:
+  Title: 更改密码
+  Description: 您的密码已成功更改。
+  NextButtonText: 继续
+
+PasswordResetDone:
+  Title: 重置链
+  Description: 请检查您的电子邮件以重置您的密码。
+  NextButtonText: 继续
+
+EmailVerification:
+  Title: 电子邮件验证
+  Description: 我们已向您发送一封电子邮件以验证您的地址。请在下面的表格中输入验证码。
+  CodeLabel: 验证码
+  NextButtonText: 继续
+  ResendButtonText: 重新发送
+
+EmailVerificationDone:
+  Title: 电子邮件验证
+  Description: 您的电子邮件地址已成功验证。
+  NextButtonText: 继续
+  CancelButtonText: 取消
+  LoginButtonText: 登录
+
+RegisterOption:
+  Title: 注册选项
+  Description: 选择您的注册方式
+  RegisterUsernamePasswordButtonText: 使用用户名密码
+  ExternalLoginDescription: 或使用外部身份提供者
+
+RegistrationUser:
+  Title: 注册
+  Description: 输入您的用户数据。您的电子邮件地址将用作登录名。
+  DescriptionOrgRegister: 输入您的用户数据。
+  EmailLabel: 电子邮箱
+  UsernameLabel: 用户名
+  FirstnameLabel: 名
+  LastnameLabel: 姓
+  LanguageLabel: 语言
+  German: 德语
+  English: 英语
+  Italian: 意大利语
+  French: 法语
+  Chinese: 简体中文
```

Comment:
please add chinese to the other language files in this folder

Reply:
Thanks for your advice, I have added Chinese to other language files.